apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx

```diff
@@ -117,15 +103,23 @@ export default function FloatingScrollbar({
 
   return (
     <div className={styles.scrollingContainer}>
-      {children}
+      <div
+        className={styles.scrollingContent}
+        ref={childContainerRef}
+        onScroll={scroll => {
+          scrollRef.current.scrollLeft = scroll.target.scrollLeft;
+        }}
+      >
+        {children}
+      </div>
       <div
         className={classNames(
           styles.scrollBar,
           !scrollVisible && styles.bottomScrollBar
         )}
         onScroll={scrollChild}
         ref={scrollRef}
-        style={{width: childWidth + 'px'}}
+        style={{width: childWidth + 'px', height: scrollbarWidth + 'px'}}
```

Comment:
There's a literal `20` up on line 68 of this file. Should it be replaced with `scrollbarWidth` too?

Reply:
Very good catch :)